ui/documents/semantic-ui/templates/documents/form/FormFieldsContainer.jsx

```diff
@@ -343,11 +360,22 @@ const FormFieldsContainer = () => {
             id="NrDocs.Deposit.NotesField.container"
             fieldPath="metadata.notes"
           >
-            <StringArrayField
+            <MultilingualTextInput
+              labelIcon="pencil"
+              label={i18next.t("Notes")}
+              textFieldLabel={i18next.t("Note")}
               fieldPath="metadata.notes"
+              rich={true}
+              editorConfig={{
+                toolbar:
+                  "bold italic | bullist numlist | outdent indent | undo redo",
+                valid_elements: "strong,b,div,br,p,i,li",
```

Comment:
List of valid tags should come from a single source of truth, do not copy it in every possible input